input/cql/Status.cql

```diff
@@ -48,7 +53,12 @@ define fluent function isLaboratoryTestOrder(ServiceRequest List<ServiceRequest>
       and S.intent = 'order'        
 
 //Diagnostic Study, Performed
-define fluent function isDiagnosticStudyPerformed(Obs List<Choice<"SimpleObservation", "LaboratoryResultObservation", "ObservationClinicalResult">>):
```

Comment:
According to the [mapping](https://hl7.org/fhir/us/qicore/STU6/qdm-to-qicore.html#diagnostic-study-performed), this should be [QICore Observation Clinical Result](https://hl7.org/fhir/us/qicore/STU6/StructureDefinition-qicore-observation-clinical-result.html) Profile

Reply:
Aren't SimpleObservation, LaboratoryResultObservation, and ObservationScreeningAssessment also considered diagnostic?